.circleci/config.yml

```diff
@@ -250,6 +268,10 @@ workflows:
       - fxa-oauth-server:
           requires:
             - install
+      - docs:
```

Comment:
Should docs follow master or tags? I don't have a preference either way

Reply:
Yeah, I can see arguments in favour of both. I like that `master` more likely reflects what I as a developer will see/use when I'm making local changes. But I can also appreciate that some people might want to see the last thing that's been released.

It wouldn't be much work to do both of course, with docs namespaced under a version number in the path if they're generated from a tag. That would have the nice effect of making it eas to go back to historical docs if ever we wanted to.

Could be a good follow-up issue for a sufficiently motivated contributor, I'll open an issue.

Reply:
Ref: #790